### DISCUSSION TITLE
What is the “version” variable for, and when should it be updated?

### DISCUSSION THREAD

**Discussion by mkavulich** [ufs-community/ufs-srweather-app#716] [Ideas]
One of the first lines in `config_defaults.yaml` is a ["version" variable](https://github.com/ufs-community/ufs-srweather-app/blob/develop/ush/config_defaults.yaml#L8). It is currently set to "1.0", and it doesn't appear to be used anywhere in the App.

Was this originally intended to be the version of the app? Some kind of “version” for controlling back-compatibility of the config file? Some other use?

If it is the first one, then it needs to be updated to 2.1 (and eventually to the next release number). And we should think about printing it in output files, xml files, and other places where it would be useful to have for user support and documentation purposes.

If it is one of the other options, then we could use some clarity on how it should be used, or if it's even necessary at all.

**Comment by natalie-perlin** [comment]
It sounds like this variable refers to "config_defaults.yaml version". Indeed seems misleading, and I'd suggest removing it to avoid going down the path of versioning all the input and configuration files. An alternative could be printing out a current SRW version of a current branch in these related configs.

**Comment by gspetro-NOAA** [comment]
The `version` variable was removed from `config_defaults.yaml` via PR #893 .
